build/common.props

```diff
@@ -1,28 +1,32 @@
 <Project>
-
   <PropertyGroup>
+    <!-- we don't want to spawn any-long living compiler processes to avoid BenchmarkDotNet issues with "file in use" -->
     <UseSharedCompilation>false</UseSharedCompilation>
-    <!-- The Python script can narrow down the TFMs to what the user has asked for -->
+    
+    <!-- to make sure that the users don't always need to specify -c Release when using dotnet run -->
+    <Optimize>true</Optimize>
+    
+    <!-- The Python script can narrow down the TFMs to what the user has asked for (to avoid downloading all SDKs) -->
```

Comment:
nit: This really is saying that we allow the python script to override the logic below. Make clear that if the Python script sets TFMs the logic below is ignored?

Reply:
Yes, this is a workaround to restore/build/run benchmark only for the specifed target framework (We only download the minimum set of runtimes, otherwise the build fails).